src/Oci8/Oci8ServiceProvider.php

```diff
@@ -58,13 +58,7 @@ public function register()
             }
 
             // set oracle session variables
-            $sessionVars = [
-                'NLS_TIME_FORMAT'         => 'HH24:MI:SS',
-                'NLS_DATE_FORMAT'         => 'YYYY-MM-DD HH24:MI:SS',
-                'NLS_TIMESTAMP_FORMAT'    => 'YYYY-MM-DD HH24:MI:SS',
-                'NLS_TIMESTAMP_TZ_FORMAT' => 'YYYY-MM-DD HH24:MI:SS TZH:TZM',
-                'NLS_NUMERIC_CHARACTERS'  => '.,',
-            ];
+            $sessionVars = config('database.connections.sessionVars');
```

Comment:
Can we set a default value? The changes might break existing installations and need to republish the config for changes to take effect.

Reply:
Yes, we can set the default